### DISCUSSION TITLE
Why is the value changed after I put it in milvus？

### DISCUSSION THREAD

**Discussion by kingzcheung** [milvus-io/milvus#18417] [Q&A and General discussion]
```go
        idColumn := entity.NewColumnInt64("id", []int64{1957642313418018817})
	productIdColumn := entity.NewColumnInt64("product_id", []int64{111})
	embeddingColumn := entity.NewColumnFloatVector("embedding", 2048, [][]float32{ebs})
	fmt.Println(productIds)
	fmt.Println(ids)
	_, err := t.client.Insert(
		ctx,
		request.GetCollName(),
		"",
		idColumn,
		productIdColumn,
		embeddingColumn,
	)
```
product_id = 111

in milvus:

![image](https://user-images.githubusercontent.com/12839855/180987627-1cdbcad1-f2b8-4dc8-a490-2f6228854457.png)

**Comment by kingzcheung** [comment]
I have updated to 2.1.0 on docker. The problem was solved.

![image](https://user-images.githubusercontent.com/12839855/180995713-4ebba0a9-dec7-4a3a-9a40-30d0287e5feb.png)
![image](https://user-images.githubusercontent.com/12839855/180995815-4af7bb62-36a7-48fe-a35a-160e6a19324f.png)

![image](https://user-images.githubusercontent.com/12839855/180995773-a4ceb217-ea16-4896-b42e-a8a53241945b.png)

It's a very strange thing

**Reply by yhmo** [reply]
Might be a bug of v2.0.2
